src/Explorer/Tabs/DocumentsTabV2/DocumentsTabV2.tsx

```diff
@@ -1915,6 +1917,26 @@ export const DocumentsTabComponent: React.FunctionComponent<IDocumentsTabCompone
     [createIterator, filterContent],
   );
 
+  /**
+   * While retrying, display: retrying now.
+   * If completed and all documents were deleted, display: all documents deleted.
+   * @returns 429 warning message
+   */
+  const get429WarningMessageNoSql = (): string => {
+    let message = 'Some delete requests failed due to a "Request too large" exception (429)';
+
+    if (bulkDeleteOperation.count === bulkDeleteProcess.successfulIds.length) {
+      message += ", but were successfully retried.";
+    } else if (bulkDeleteMode === "inProgress" || bulkDeleteMode === "aborting") {
+      message += "(429). Retrying now.";
```

Comment:
I don't think you need the `(429)` prefix here, it's already in the base message.

```suggestion
      message += ". Retrying now.";
```